CHANGELOG.md

```diff
@@ -65,6 +65,12 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### New features
 
+* Define `immutable_heads()` revset alias in terms of a new `builtin_immutable_heads()`.
+  This enables users to redefine `immutable_heads()` as they wish, but still
+  have `builtin_immutable_heads()` which can should not be redefined. Also 
+  issue warnings if any of `mutable()`, `immutable()` or `builtin_immutable_heads()`
+  are redefined by the user.
```

Comment:
nit: `which can should` -> `which should`

And I would remove the last "Also issue warnings ..." sentence because user wouldn't have to know that beforehand.

Reply:
Whoops. Thanks for catching. Also removed last sentence.